test/e2e/common/node/lifecycle_hook.go

```diff
@@ -93,6 +94,15 @@ var _ = SIGDescribe("Container Lifecycle Hook", func() {
 				},
 			}
 			podWithHook := getPodWithHook("pod-with-poststart-exec-hook", imageutils.GetE2EImage(imageutils.Agnhost), lifecycle)
+			// make sure we spawn the test pod on a different node than the webserver.
```

Comment:
on a different or in the same node?, I though that we had to use the same node

Reply:
Oups, copied the comment from the tests that were enforcing the nodes to be different, didn't update the comment.